Sources/SwiftSyntax/gyb_generated/SyntaxFactory.swift

```diff
@@ -626,15 +626,15 @@ public enum SyntaxFactory {
     let raw = RawSyntax.createAndCalcLength(kind: SyntaxKind.floatLiteralExpr,
       layout: layout, presence: SourcePresence.present)
     let data = SyntaxData.forRoot(raw)
-    return FloatLiteralExprSyntax(data)
+    return FloatLiteralExprSyntax(data)!
```

Comment:
This crashes if the argument `floatingDigits` is invalid.
Could you make this `makeFloatLiteralExpr` and `makeIntegerLiteralExpr` failable (i.e. return `Optional`)?

Reply:
Done. Getting the Python indentation in `SyntaxFactory.swift.gyb` right was tricky. 😅